.circleci/config.yml

```diff
@@ -266,23 +269,98 @@ jobs:
           path: /tmp/license-scanner/results
           prefix: licenses
 
-  build:
+  build-local:
     executor: default-machine
     steps:
       - checkout
       - run:
-          name: Build Docker $CIRCLE_TAG image
+          name: Build Docker local image
           command: |
-            echo "Building Docker image: $CIRCLE_TAG"
-            docker build -t $DOCKER_ORG/$CIRCLE_PROJECT_REPONAME:$CIRCLE_TAG .
+            echo "Building Docker image: local"
+            docker build -t $DOCKER_ORG/$CIRCLE_PROJECT_REPONAME:local .
       - run:
           name: Save docker image to workspace
-          command: docker save -o /tmp/docker-image.tar $DOCKER_ORG/$CIRCLE_PROJECT_REPONAME:$CIRCLE_TAG
+          command: docker save -o /tmp/docker-image.tar $DOCKER_ORG/$CIRCLE_PROJECT_REPONAME:local
       - persist_to_workspace:
           root: /tmp
           paths:
             - ./docker-image.tar
 
+  release:
+    executor: default-docker
+    steps:
+      - checkout
```

Comment:
Maybe move this to after the install dependencies to keep things consistent

Reply:
going to move `checkout` to before dependency install instead.

Reply:
Nah you don't want to do that. I had it like that earlier, but if git isn't installed, circleci defaults to a bad version of git which can't checkout tags

Reply:
good to know. done. added a comment as well.